Berksfile

```diff
@@ -2,39 +2,32 @@ source 'https://supermarket.chef.io'
 
 # OSL Base deps
 cookbook 'aliases', git: 'git@github.com:osuosl-cookbooks/aliases'
-cookbook 'apt', '>= 2.3.8'
-cookbook 'apache2', '< 2.0.0'
 cookbook 'base', git: 'git@github.com:osuosl-cookbooks/base'
-cookbook 'database', '>= 2.0.0'
 cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
 cookbook 'modules', git: 'git@github.com:osuosl-cookbooks/modules-cookbook'
 cookbook 'monitoring', git: 'git@github.com:osuosl-cookbooks/monitoring'
 cookbook 'munin'
-cookbook 'nagios', git: 'git@github.com:osuosl-cookbooks/nagios'
+cookbook 'osl-nagios', git: 'git@github.com:osuosl-cookbooks/osl-nagios'
+cookbook 'osl-nrpe', git: 'git@github.com:osuosl-cookbooks/osl-nrpe'
```

Comment:
My intention is that this PR is depended on that but I don't expect this to be ready for deploying to production for another month or so. So I'm not that concerned.